### ISSUE TITLE
The cache directory is not created, thus nuts fails

### ISSUE THREAD

**Issue by kumarharsh** [GitbookIO/nuts#41]
When I try to use nuts, it throws an error on creating the cache directory, and throws this:

```
Error: EPERM: operation not permitted, stat 'C:\Users\khs\AppData\Local\Temp\nuts\6441f29b52975b97bf3c6b7eb9fb2e8341c28632'
```

Basically, the `nuts` directory does not exist, but the LRU-diskcache is trying to create a folder inside it. Whose responsibility should it be to check that the parent folder is created? LRU's or nuts' ?

**Comment by nahueld** [comment]
same happens in heroku

**Comment by kumarharsh** [comment]
Yes, it'll happen because of the reason I specified - the `nuts` directory inside the `/tmp` does not exist. You have to create it manually. You can do this just before initializing nuts by checking for the directory, and running `fs.mkdir`

**Comment by AaronO** [comment]
@kumarharsh @nahueld This is now fixed in `3.0.0-pre.6`, since `nuts` now automatically calls `.init()`, you don't need to call it yourself. Don't do a manual `fs.mkdir()` because `Nuts.init()` may do more
